GrailDiary/GrailDiary/ViewControllers/POIsTableViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ViewController.swift
+//  GrailDiary
+//
+//  Created by Violet Lavender Love on 5/12/20.
+//  Copyright © 2020 Love. All rights reserved.
+//
+
+import UIKit
+
+class POIsTableViewController: UIViewController {
+    @IBOutlet var POIsTableView: UITableView!
+    
+    var manyPOIs: [POI] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+}
+
+
+extension POIsTableViewController: UITableViewDataSource {
+   
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        
+        return manyPOIs.count
+        
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "POICell", for: indexPath) as? POITableViewCell else {
+            fatalError("Cell is not of expected identifier POICell. If you are a user, I am very sorry. Please restart your app!")
```

Comment:
You're close. If this were to fail, we would want to return an empty UITableViewCell instead of a "fatalError". So how would we do that?